R/dataProcessPlots.R

```diff
@@ -129,20 +135,91 @@ dataProcessPlots = function(
     }
   }
   
-  if (type == "PROFILEPLOT") 
-    .plotProfile(processed, summarized, featureName, ylimUp, ylimDown,
-                 x.axis.size, y.axis.size, text.size, text.angle, legend.size, 
-                 dot.size.profile, width, height, which.Protein, originalPlot, 
-                 summaryPlot, remove_uninformative_feature_outlier, address)
-  if (type == "QCPLOT")
-    .plotQC(processed, featureName, ylimUp, ylimDown, x.axis.size, y.axis.size, 
-            text.size, text.angle, legend.size, dot.size.profile, width, height,
-            which.Protein, address)
-  if (type == "CONDITIONPLOT")
-    .plotCondition(processed, summarized, ylimUp, ylimDown, scale, interval,
-                   x.axis.size, y.axis.size, text.size, text.angle, legend.size, 
-                   dot.size.profile, dot.size.condition, width, height,
-                   which.Protein, save_condition_plot_result, address)
+  if(isPlotly & address != FALSE) {
+      print("Plots will be saved as .HTML file as plotly is selected, set isPlotly = FALSE, if 
```

Comment:
Rather than printing, should we throw an exception to the user and halt execution in this case?

Reply:
if address is not FALSE, some file will saved, right? So the function should just continue. Or am I missing something?

Reply:
Got it, I took a second look at the code again and continuing should be fine, but I think the parameter description on line 68-69 is a bit misleading.

Maybe modify the parameter description in line 68-69 from "address and isPlotly cannot be set as TRUE at the same time."  to something like "address and isPlotly can both be set to true, but MSstats will save HTML files as a result."

Reply:
@tonywu1999, thanks for pointing it out! I have updated the documentation here [better documentation on parameters](https://github.com/Vitek-Lab/MSstats/pull/107/commits/9e51b08eb9924babe0fb7b7d8dd695ae80b9f38d)

Hope this makes better sense